File: translations/codenet/deepseek_commented/Go2C++/codellama/Code/s695527706.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	int n; // Declaring an integer variable to store the number of inputs
	string s; // Declaring a string variable to store the input string
	cin >> n >> s; // Reading input from the user and storing it in n and s
	cout << count(s.begin(), s.end(), 'A') << endl; // Counting the occurrences of 'A' in s and printing the result
	return 0;
}

File: translations/codenet/deepseek_commented/Go2P/gpt-4o/Code/s695527706.py
# Importing the necessary module for string manipulation
import sys

def main():
    # Reading input from the user
    n, s = sys.stdin.read().splitlines()
    n = int(n)  # Converting the first line to an integer
    # Counting the occurrences of "ABC" in s and printing the result
    print(s.count("ABC"))

if __name__ == "__main__":
    main()

# <END-OF-CODE>
